### ISSUE TITLE
transformClassesWithDexBuilderForDebug FAILED

### ISSUE THREAD

**Issue by projetoarduino** [oney/react-native-webrtc#526]
* **React Native version**:
react-native-cli: 2.0.1
react-native: 0.57.3

* **Plugin version**:
"react-native-webrtc": "^1.67.1"

* **OS**:
Android
* **OS version**:
Api 27

Hi follow my steps

react-native init  webandroidrtc
cd webandroidrtc/
npm install --save react-native-webrtc@1.67.1
 react-native link react-native-webrtc
 react-native run-android

And i had this compilation error

 Task :app:transformClassesWithDexBuilderForDebug FAILED

**Comment by saghul** [comment]
Have you enabled Java 8 support, as the documentation indicates?

**Comment by saghul** [comment]
That’s not correct. You are now using an old and unsupported version. I do not recommend you do that, but it’s ultimately up to you.

**Comment by projetoarduino** [comment]
Please help me solve
